test/mock/kernel/vga_mock.zig

```diff
@@ -69,11 +69,11 @@ pub fn init() void {
 
 // User defined mocked functions
 
-pub fn mock_entryColour(fg: u4, bg: u4) u8 {
+pub fn orig_entryColour(fg: u4, bg: u4) u8 {
```

Comment:
What's up with the renaming?

Reply:
I documented in one of the wiki files that if the mocked function has the same behavior as the source function, then prefix with `orig_`.

Reply:
I see